Bottom: d401d2837159c4896c49d73031c3a31d052ef477
Top:    7226508d705d86bb7d601bab8dd5cb56ed817ead
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:26 +0000

Disable debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 364d875..ba5c857 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1504,7 +1504,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                               copper_appearance,
                               NULL));
 
-    break;
+    //break;
   }
 
 
@@ -1516,10 +1516,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 //  gui->invalidate_all ();
 
   /* ASSUME THERE IS A POLYGON WHERE WE KNOW WE PUT ONE... */
-  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
-  gui->invalidate_all ();
+//  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
+//  gui->invalidate_all ();
 
-//  poly_Free (&info.poly);
+  poly_Free (&info.poly);
 
   return objects;
 }
